### ISSUE TITLE
Complement operator is not supported

### ISSUE THREAD

**Issue by marcospassos** [ApiGen/ApiGen#720]
ApiGen is unable to generate documentation for any code that uses PHP's two's-complement operator.

```
Writing lock file
Generating autoload files
bin/apigen generate -d api/latest
Scanning sources and parsing
Parse error: "Value definition is empty."
```

**Comment by jadb** [comment]
Sorry for the delay.

This is very interesting. Have you been able to fix it?

**Comment by POPSuL** [comment]
I guess this trouble is related to PHP 5.6 const arrays and it was fixed in #726

**Comment by jadb** [comment]
I am closing this then. Please submit new issue if it's still broken for you @marcospassos.
